test/Altinn.AccessManagement.Tests/Data/Json/MaskinportenSchema/DelegationCheck/scope-access-schema/from_p50004221/authn_u20000490.json

```diff
@@ -0,0 +1,206 @@
+[
+  {
+    "rightKey": "generic-access-resource:read",
+    "resource": [
+      {
+        "id": "urn:altinn:resource",
+        "value": "generic-access-resource"
+      }
+    ],
+    "action": "read",
+    "status": "NotDelegable",
+    "details": [
+      {
+        "code": "MissingRoleAccess",
+        "description": "Delegator does not have any required role(s) for the reportee party: (urn:altinn:rolecode:PRIV, urn:altinn:rolecode:DAGL), which would give access to delegate the right.",
+        "parameters": {
+          "RequiredRoles": "urn:altinn:rolecode:PRIV, urn:altinn:rolecode:DAGL"
```

Comment:
These tests will fail once @ivartryti's PR is merged which change the response model for parameters to List<AttributeMatch>. 
The race is on! Who will get their PR merged first and avoid the conflict and need to update the tests? :P